aries-site/src/pages/templates/popover.mdx

```diff
@@ -0,0 +1,55 @@
+import { Example } from '../../layouts';
+import { PopoverAnatomy, PopoverSimpleExample, PopoverInlineExample } from '../../examples';
+
+<Example
+  figma="https://www.figma.com/design/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?node-id=34141-2"
+  width="100%"
+>
+  <PopoverSimpleExample />
+</Example>
+
+## Use cases
+
+### Providing inline contextual help
+
+A Popover may be triggered through an icon-only button to provide the user with helpful contextual info in a non-intrusive manner.
+
+<Example
+  figma="https://www.figma.com/design/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?node-id=34141-2"
+  width="100%"
+>
+  <PopoverInlineExample />
+</Example>
+
+## Anatomy
+
+<Example caption="Diagram illustrating the 4 parts of a Popover." plain>
+  <PopoverAnatomy />
+</Example>
+
+| Label  | Region                  | Purpose                               | Required | Notes                                               |
+| :----: | ----------------------- | ------------------------------------- | :------: | --------------------------------------------------- |
+| **1**  | **Title**               | Identifies the Popover.               | Optional |                                                     |
+| **2**  | **Close Button**        | Closes the Popover.                   | Required | --                                                  |
+| **3**  | **Body**                | Provides contextual information.      | Required | May include other elements that are not text.       |
+| **4**  | **Footer**              | Contains additional Popover actions.  | Optional | May contain anchors.                                |
+
+## Content
+Popovers should be:
+- Relevant - The Popover should relate to its trigger and provide additional contextual information.
+- The content should not contain crucial information or actions required to complete a task.
+- Short -  Messages should be simple, too the point, and small to avoid overloading the user and taking up too much screen real estate. Follow the voice and tone guidelines for writing crisp and effective content.
+
+It is recommended that Popovers:
+- Use Paragraph/Small and Anchor/Small to save real estate.
+- Avoid using titles if it provides no additional context.
+
+## Accessibility
```

Comment:
In this section should we mention we don't recommend using headings for titles?

Reply:
yes we can add let me chat with Kenny

Reply:
added